scripts/python/junglecatsoftware/awstoolkit/outputFormatters.py

```diff
@@ -24,3 +25,8 @@ def printStartDateStamp():
 def printIndented( text, indentLevel = 1 ):
   for line in text.splitlines():
     print( ( '\t' * indentLevel ) + line )
+
+if __name__ == "__main__":
```

Comment:
Ah, testing! I like it! I might actually add more to this later, but I think this might be a habit I continue with for other things.